screens/EpisodeInputScreen.tsx

```diff
@@ -113,14 +104,29 @@ export default function EpisodeInputScreen() {
 
   const [episodes, setEpisodes] = useState<Episode[]>([]);
 
-  const putEpisodesInStorage = async (finished : boolean) => {
+  const putEpisodesInStorage = async (finished: boolean) => {
     await AsyncStorage.setItem(
       STORAGE_KEYS.daysEpisodes(recordingDay),
       JSON.stringify(episodes)
     );
     if (finished) {
       mainNavigation.navigate({ type: "recordEpisodes", episodes });
+      const randomEpisodes = selectRandomEpisodes();
+      await AsyncStorage.setItem(
+        STORAGE_KEYS.episodeRecall(),
+        JSON.stringify(randomEpisodes)
+      );
+    }
+  };
+
+  const selectRandomEpisodes = () => {
+    const randomEpisodeInx = () => Math.floor(Math.random() * episodes.length);
+    const firstElement = randomEpisodeInx();
+    let secondElement = randomEpisodeInx();
+    while (firstElement === secondElement) {
+      secondElement = randomEpisodeInx();
     }
```

Comment:
Saving the episodes that they need to recall immediately upon their creation on Day 1. I didn't even think about it as a possibility, but darn it's genius

Reply:
The fun thing about this is that there IS a bug here (but it is a bit hidden). This saves episodes on EVERY day that you record episodes, not only the first one. This will be fixed, we apologize and BEG for your forgiveness